gestalt/vault.py

```diff
@@ -1,29 +1,55 @@
+import os
 from datetime import datetime, timedelta
+from queue import Queue
+from threading import Thread
 from time import sleep
-from gestalt.provider import Provider
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import hvac  # type: ignore
 import requests
-from requests.exceptions import Timeout
 from jsonpath_ng import parse  # type: ignore
-from typing import Optional, Tuple, Any, Dict, Union, List
-import hvac  # type: ignore
-from queue import Queue
-import os
-from threading import Thread
-from retry import retry
+from requests.exceptions import Timeout
+from retry.api import retry_call
+
+from gestalt.provider import Provider
 
 
 class Vault(Provider):
-    @retry((RuntimeError, Timeout), delay=2, tries=5)  # type: ignore
-    def __init__(self,
-                 cert: Optional[Tuple[str, str]] = None,
-                 role: Optional[str] = None,
-                 jwt: Optional[str] = None,
-                 url: Optional[str] = os.environ.get("VAULT_ADDR"),
-                 token: Optional[str] = os.environ.get("VAULT_TOKEN"),
-                 verify: Optional[bool] = True,
-                 scheme: str = "ref+vault://") -> None:
+    def __init__(
+        self,
+        cert: Optional[Tuple[str, str]] = None,
+        role: Optional[str] = None,
+        jwt: Optional[str] = None,
+        url: Optional[str] = os.environ.get("VAULT_ADDR"),
+        token: Optional[str] = os.environ.get("VAULT_TOKEN"),
+        verify: Optional[bool] = True,
+        scheme: str = "ref+vault://",
+        delay: int = 2,
+        tries: int = 5,
+    ) -> None:
+        self.delay = delay
+        self.tries = tries
+
+        retry_call(
```

Comment:
this is not pythonic

Reply:
While I'm going to revert this, could you expand on what isn't pythonic here for future reference?

Reply:
Got it, appreciate the clarification!